specification/2.0/README.md

```diff
@@ -2026,10 +1963,10 @@ A buffer points to binary geometry, animation, or skins.
 |   |Type|Description|Required|
 |---|----|-----------|--------|
 |**uri**|`string`|The uri of the buffer.|No|
-|**byteLength**|`integer`|The total byte length of the buffer view.| :white_check_mark: Yes|
```

Comment:
This is worded better in the schema, and even contains a fix: This is a buffer, not a buffer view.